src/static/js/modules/skip-nav.js

```diff
@@ -0,0 +1,20 @@
+/* ==========================================================================
+   Skip Nav
+
+   This is necessary because of a webkit quirk with handling keyboard
+   focus with anchor links. If that webkit bug is fixed, remove this JS.
+
+========================================================================== */
+
+'use strict';
+
+var $ = require( 'jquery' );
+
+function init() {
+  $( 'a[href^="#"]' ).click( function() {
```

Comment:
Since this is for any `:target` link, the file should probably be named something like `focus-target.js`.